s9/loop_meth.rb

```diff
@@ -0,0 +1,21 @@
+name = %w(H I J K L)
+# break
+i=0
+while i<name.length
+  puts "#{i}. Nombre: #{name[i]}"
+  i+=1
+  if name[i]=="K"
+    break
+  end
+end
+puts
+
+# next
+name.each do |n|
```

Comment:
What happens if `name` is empty?

Reply:
If the name variable were empty, Each method would not do any iteration since the Array has nothing to go through.